subgraphs/rocket-pool/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "license": "MIT",
+  "scripts": {
+    "format": "npx prettier --write ."
+  },
+  "dependencies": {
+    "@graphprotocol/graph-cli": "0.36.0",
```

Comment:
might as well update these to the latest versions

Reply:
updated